args/physics/PhysicsStatics.h

```diff
@@ -0,0 +1,91 @@
+#pragma once
+#include <physics/physicsimport.h>
+#include <physics/convexcollider.hpp>
+
+namespace args::physics
+{
+	struct HalfEdgeFace;
+
+	class PhysicsStatics
+	{
+	public:
+
+        /** @brief Given a transformed ConvexCollider and a direction, Gets the vertex furthest in the given direction
+        * @param planePosition The position of the support plane in world space
+        * @param direction The direction we would like to know the support point of
+        * @param collider The ConvexCollider in question
+        * @param colliderTransform A mat4 describing the transform of the collider
+        * @param worldSupportPoint [out] the resulting support point
+        */
+        static void GetSupportPoint(math::vec3 planePosition, math::vec3 direction, ConvexCollider* collider, math::mat4 colliderTransform
+            ,math::vec3& worldSupportPoint)
+        {
+            float largestDistanceInDirection = -FLT_MAX;
```

Comment:
consider replacing this with `std::numeric_limits<float>::lowest()`

Reply:
if anything at least FLT_MIN, since -FLT_MAX and FLT_MIN are not the same. FLT_MIN is actually: 1.175494e-38 and FLT_MAX is: 3.402823e+38